src/python/pants/backend/jvm/subsystems/zinc_language_mixin.py

```diff
@@ -27,7 +27,8 @@ def register_options(cls, register):
 
     register('--compiler-option-sets', advanced=True, default=[], type=list,
              fingerprint=True,
-             help='The option sets to be enabled for the compilation of this target.')
+             help='The default for the "compiler_option_sets" argument'
```

Comment:
String concatenation won't leave a space here. We generally use a trailing space on the first line.

Reply:
(I went ahead and used the editing UI to fix this. That's nifty.)